Makefile

```diff
@@ -133,7 +138,7 @@ endif
 .PHONY: clean all test lint doc clean_all rcpplint rcppexport roxygen\
 	cython2 cython3 cython cyclean
 
-all: lib/libmxnet.a lib/libmxnet.so $(BIN)
+all: lib/libmxnet.a lib/libmxnet.so $(BIN) cpp-package-example-all
```

Comment:
It only generates a header file for cpp-package, and is necessary since updates that break cpp-package will be reflected in the test..